tasks/Module 3/task_1.py

```diff
@@ -0,0 +1,36 @@
+"""Sandwich calculator."""
+import pyinputplus as pyip
+
+BREAD_COST = {"wheat": 1, "white": 1.5, "sourdough": 1.5}
+MEAT_COST = {"chicken": 3, "turkey": 4, "ham": 3, "tofu": 4}
+CHEESE_COST = {"cheddar": 2, "swiss": 1.5, "mozzarella": 1}
+SAUSE_COST = {"mayo": 0.2, "mustard": 0.1, "lettuce": 0.2, "tomato": 0.1}
+
+
+def sandwich_maker():
+    """Build sandwich."""
+    print("Welcome to the sandwich ordering program!!! Please choose ingridients for your sandwich:")
+    print(' BREAD '.center(20, '='))
```

Comment:
Smart solution, I like it :)

Reply:
How about creating a helper function, eg `print_section_header` taking a string, eg. `BREAD` and printing it in such a nice way?